net/portmapper/igd_test.go

```diff
@@ -49,10 +51,10 @@ func NewTestIGD() (*TestIGD, error) {
 		doUPnP: true,
 	}
 	var err error
-	if d.upnpConn, err = net.ListenPacket("udp", "127.0.0.1:1900"); err != nil {
+	if d.upnpConn, err = netns.Listener().ListenPacket(context.Background(), "udp4", ":1900"); err != nil {
```

Comment:
this will be angry at macOS with its firewall pop-up dialogs. we really don't want to listen on non-localhost on macos.

Reply:
I'm on mac, but haven't noticed anything from this?

Reply:
Depends on settings. You might have it off.

<img width="780" alt="Screen Shot 2021-08-09 at 2 18 00 PM" src="https://user-images.githubusercontent.com/2621/128775850-31e4eb4c-ec62-4d51-8d87-c73ab7aa4d67.png">
<img width="780" alt="Screen Shot 2021-08-09 at 2 18 07 PM" src="https://user-images.githubusercontent.com/2621/128775856-a911a62a-e785-46e8-8d88-a8b00111a341.png">

Reply:
ah you're right the firewall is off